config/testgrids/kubernetes/presubmits/config.yaml

```diff
@@ -144,6 +144,9 @@ dashboards:
     base_options: width=10
   - name: pull-kubernetes-node-e2e
     test_group_name: pull-kubernetes-node-e2e
+  - name: pull-kubernetes-node-crio-e2e
```

Comment:
/approve
for tracking here, though I recommend prow.k8s.io for this purpose (presubmits do not tend to do well in testgrid)
the other changes I think should be reviewed by a sig node reviewer